lib/seadsa/DsaLibFuncInfo.cc

```diff
@@ -14,12 +14,23 @@
 #include "seadsa/Graph.hh"
 #include "seadsa/InitializePasses.hh"
 #include "seadsa/support/Debug.h"
+#include <filesystem>
 #include <iostream>
 
 static llvm::cl::list<std::string>
     XSpecFiles("sea-dsa-specfile", llvm::cl::desc("<Input spec bitcode file>"),
                llvm::cl::ZeroOrMore, llvm::cl::value_desc("filename"));
 
+static llvm::cl::opt<bool>
+    XUse32BitSpec("sea-dsa-use-32bit-specs",
```

Comment:
32 vs 64 bit should be decided based on current `DataLayout`. You can grab a `DataLayout` from `Module` and check whether current pointer width is `32` or `64` bits